graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java

```diff
@@ -261,6 +292,23 @@ private GraphQLType buildFilterInput(Table table) {
         .build();
   }
 
+  private GraphQLType buildGroupByInput(Table table) {
+    GraphQLInputObjectType.Builder builder =
+        GraphQLInputObjectType.newInputObject()
+            .name(nameMapping.getGraphqlName(table) + "GroupByInput");
+    for (Column column : table.primaryKeyColumns()) {
```

Comment:
should this be only for clustering keys?

Reply:
I had to double check, but no, it's also possible to group by partition key columns (although the query will generally involve filtering).